Changelog.md

```diff
@@ -1,5 +1,63 @@
 ## Gazebo 11
 
+## Gazebo 11.13.0 (2023-05-17)
+
+1. Fix build with ffmpeg 6.0
+   * [Pull request #3318](https://github.com/gazebosim/gazebo-classic/pull/3318)
+
+1. Allow user to name a specific light that will generate lens flare
+   * [Pull request #3305](https://github.com/gazebosim/gazebo-classic/pull/3305)
+   * A contribution from Terry Welsh 
+
+1. JointController: improve thread safety
+   * [Pull request #3303](https://github.com/gazebosim/gazebo-classic/pull/3303)
+
+1. Fix pkg-config-related CMake warning
+   * [Pull request #3295](https://github.com/gazebosim/gazebo-classic/pull/3301)
+
+1. Fix template specialization in constructors to fix GCC11 build
+   * [Pull request #3295](https://github.com/gazebosim/gazebo-classic/pull/3295)
+   * A contribution from Ondřej Svoboda
+
+1. Set initial sim time from the command-line
+   * [Pull request #3294](https://github.com/gazebosim/gazebo-classic/pull/3294)
+
+1. Fix typo in README
+   * [Pull request #3291](https://github.com/gazebosim/gazebo-classic/pull/3291)
+
+1. msgs.cc: add missing <array> include
+   * [Pull request #3290](https://github.com/gazebosim/gazebo-classic/pull/3290)
+
+1.  Fix crash with DEM and Camera
+   * [Pull request #3279](https://github.com/gazebosim/gazebo-classic/pull/3279)
+
+1.  Fix focal builds: use python3 with check_test_ran.py
+   * [Pull request #3278](https://github.com/gazebosim/gazebo-classic/pull/3278)
```

Comment:
this targeted `gazebo9`, so it should be removed from here

Reply:
Done!